scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -417,7 +458,9 @@ task M2 {
       gatk --java-options "-Xmx~{command_mem}m" Mutect2 \
         -R ~{ref_fasta} \
         -I ~{input_bam} \
-        ~{"--alleles " + gga_vcf} \
+        ~{"--genotyping-mode GENOTYPE_GIVEN_ALLELES --alleles " + gga_vcf} \
```

Comment:
I'm confused, I thought we got rid of the "genotyping-mode" argument in later versions not the other way around. Can you try running this with a gga_vcf just to make sure it doesn't throw a user error? I don't think we're testing this argument in the WDL tests, but you could also add it there (it's left out now because it's an optional parameter).

Reply:
removing this argument all together. it can be added to the m2_extra_args if needed